core/src/main/java/org/opensearch/sql/expression/function/OpenSearchFunctions.java

```diff
@@ -86,10 +93,19 @@ private static FunctionResolver wildcard_query(BuiltinFunctionName wildcardQuery
     return new RelevanceFunctionResolver(funcName);
   }
 
+  private static FunctionResolver score(BuiltinFunctionName score) {
+    FunctionName funcName = score.getName();
+    return new RelevanceFunctionResolver(funcName);
```

Comment:
Why use `RelevanceFunctionResolver` instead of `DefaultFunctionResolver`?

`RelevanceFunctionResolver` is needed because parameter validation for relevance function happens in corresponding RelevanceQuery-derived class.

Reply:
I used the RelevanceFunctionResolver because I need an OpenSearchFunction back, but the resolver isn't really used.  It's more of a convenience to use the RelevanceFunctionResolver even-though nothing really gets resolved (only the child gets resolved)